blackboxopt/optimization_loops/sequential.py

```diff
@@ -53,9 +54,12 @@ def run_optimization_loop(
             exception raised by the evaluation function or instead store their stack
             trace in the evaluation's `stacktrace` attribute. Set to True if there are
             spurious errors due to e.g. numerical instability that should not halt the
-            optimization loop.
+            optimization loop. For more details, see the wrapper that is used internally
+            `blackboxopt.optimization_loops.utils.evaluation_function_wrapper`
+        pre_evaluation_callback: Reference to a callable that is invoked before each
```

Comment:
I'm wondering: What's actually the use case for this pre-evaluation callback? And if I'm asking myself this question, might it make sense to add such an example to the docstring, too?

Reply:
@LGro, makes sense, thanks!